components/sample/Shell_script/component.yaml

```diff
@@ -0,0 +1,25 @@
+name: Filter text
```

Comment:
Shall we name this grep? I think people want to know the pattern is a grep pattern

Reply:
I'm not 100%.
We have 4 samples doing the filtering in Shell, Python, C# and R (need Java) and they all have same name to highlight they're doing the same thing.
Perhaps `Filter text with shell and grep`.

Reply:
Renamed